source/agora/test/Simple.d

```diff
@@ -28,7 +28,11 @@ unittest
 {
     TestConf conf;
     conf.consensus.payout_period = 10;
+    simpleTest(conf);
+}
 
+public void simpleTest (TestConf conf)
```

Comment:
Why don't you add this function to Base.d with the more descriptive name because it would be better to have no relation between test modules, I guess?

Reply:
I am not sure if importing from simple test module is so bad when I just want a simple test but with a different test configuration. Maybe others can also comment.